### ISSUE TITLE
[Bug]:  Deployable docker compose missing `config` property

### ISSUE THREAD

**Issue by flexchar** [coollabsio/coolify#2802]
### Description

A source docker compose file with top-level `config` property is gone at deployment version.

Docs: https://docs.docker.com/compose/compose-file/08-configs/

### Minimal Reproduction (if possible, example repository)

Imagine this source `docker-compose.yaml`

```yaml
services:
  litellm:
    image: ghcr.io/berriai/litellm:main-latest
    environment:
      - OPENAI_API_KEY=${OPENAI_API_KEY}
      - NUM_WORKERS=8
    configs:
      - source: litellm-config
        target: /app/config.yaml

configs:
  litellm-config:
    # https://docs.litellm.ai/docs/proxy/configs
    content: |
      model_list:
        # OpenAI
        - model_name: gpt-4o
          litellm_params:
            model: openai/gpt-4o
            api_key: 'os.environ/OPENAI_API_KEY'
```

### Exception or Error

service "litellm" refers to undefined config litellm-config: invalid compose project
(as a result of missing `configs`)

### Version

v4.0.0-beta.309

**Comment by andrasbacsai** [comment]
This will be fixed in the upcoming version.

**Comment by andrasbacsai** [comment]
It was released in v4.0.0-beta.311, just forgot to close this.
